### DISCUSSION TITLE
IMU Subscribe Error after 22.04 clean install at HSR-C (also B)

### DISCUSSION THREAD

**Discussion by ghost** [HumanSupportRobot/QandA#22] [Q&A] [answered]
I installed 22.04 HSR software to our HSR-C (S/N: 51) with clean install image.
After successfully Install of 22.04, we checked the devices with diagnostic tools,
and `imu_sensor_check` failed with `Subscribe Error`.

![Screenshot from 2022-10-20 13-14-43](https://user-images.githubusercontent.com/71804530/196857400-a46cd528-65f3-4587-8e05-52b7fb14e3a7.png)

However, move_base appears to be working correctly,
because omni base of HSR-C was moved by `2D Nav Goal` topic from rviz.

I want all errors are resolved. So, could you give me some hints, please?

**Comment by datemitumasa** [comment]
We installed 22.04 HSR software(hsrb-installer_22.04.01_focal_HSRB-PHASE5_ja_JP.UTF-8.iso) to our HSR-B (S/N: 105) with clean install image yesterday too.The same problem occurs here as well.
We checked and there is no "/hsrb(or hsrc)/base_imu/data", is it the same?

**Reply by ghost** [reply]
Yes, same.
Our HSR-C also doesn't publish `/hsrb/base_imu/data` topic.

**Reply by hsr-admin** [reply]
We apologize for any inconvenience caused.
This bug will be fixed in the soon-to-be-released software (22.10), so please be patient.

**Reply by ghost** [reply]
Thank you for your answer.
(@hsr-admin But could you re-answer as a new "conversation" in this discussion in order to "Mark as answer".)

**Reply by datemitumasa** [reply]
@hsr-admin 
Is there any negative impact on path plan?

**Reply by hsr-admin** [reply]
>Is there any negative impact on path plan?

IMU data is not used for navigation, so IMU has no effect on the path plan.

**Comment by hsr-admin** [comment, marked as answer]
> (@hsr-admin But could you re-answer as a new "conversation" in this discussion in order to "Mark as answer".)

We apologize for any inconvenience caused.
This bug will be fixed in the soon-to-be-released software (22.10), so please be patient.
